README.md

```diff
@@ -67,14 +65,9 @@ Console using `msbuild`. For instance:
 msbuild UIAutomation.sln /p:Configuration=Release,Platform=x64
 ```
 
-Should you install a newer SDK, you will need to re-target the projects to that version. (The simplest way
-is to search for `19613` in all `.vcxproj` files and update them to the new version.) Note, however,
-that there are no guarantees that the platform API will be unchanged and as such the project might not
-build or it could work differently at runtime.
-
-In the future, once the platform API has stabilized into an official SDK release, this project will
-target the stable version.
-
+As  these projects have been written against an older preview sdk (10.0.19613.0), there are no guarantees that the platform API will be unchanged and as such the project might not
+build or it could work differently at runtime on newer versions.
```

Comment:
I'm not sure this comment makes sense anymore. Maybe say something like "These projects target SDK version 10.0. Note that breaking changes can occur in preview SDKs, so if you install a new preview SDK there are no guarantees that the platform API will be unchanged and as such the projects might not build or they could work differently at runtime."